mne/utils.py

```diff
@@ -752,6 +752,35 @@ def _check_mayavi_version(min_version='4.3.0'):
         raise RuntimeError("Need mayavi >= %s" % min_version)
 
 
+def _check_pyface_backend():
+    """Check the currently selected Pyface backend
+
+    Returns
+    -------
+    backend : str
+        Name of the backend.
+    result : 0 | 1 | 2
+        0: the backend has been tested and works.
+        1: the backend has not been tested.
+        2: the backend not been tested.
```

Comment:
Sorry this is just a place holder PR for an issue, supposed to be "does not work"


> On Nov 3, 2015, at 12:18 PM, jaeilepp <notifications@github.com> wrote:
> 
> In mne/utils.py <https://github.com/mne-tools/mne-python/pull/2585#discussion_r43777452>:
> 
> > @@ -752,6 +752,35 @@ def _check_mayavi_version(min_version='4.3.0'):
> >          raise RuntimeError("Need mayavi >= %s" % min_version)
> >  
> >  
> > +def _check_pyface_backend():
> > +    """Check the currently selected Pyface backend
> > +
> > +    Returns
> > +    -------
> > +    backend : str
> > +        Name of the backend.
> > +    result : 0 | 1 | 2
> > +        0: the backend has been tested and works.
> > +        1: the backend has not been tested.
> > +        2: the backend not been tested.
> What's the difference between 1 and 2?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mne-tools/mne-python/pull/2585/files#r43777452>.
>